web/js/palettes/palettes.model.js

```diff
@@ -1,26 +1,26 @@
-/**
- * @module wv.palette
- */
-var wv = wv || {};
-wv.palettes = wv.palettes || {};
-
-/**
- * @class wv.palette.model
- */
-wv.palettes.model = wv.palettes.model || function (models, config) {
+import each from 'lodash/each';
+import find from 'lodash/find';
+import cloneDeep from 'lodash/cloneDeep';
+import isNaN from 'lodash/isNaN';
+import isUndefined from 'lodash/isUndefined';
+import parseInt from 'lodash/parseInt';
+import util from '../util/util';
+import palettes from '../palettes/palettes';
+
+export function palettesModel(models, config) {
```

Comment:
What do you think about keeping the name `model`, importing this into the `palettes` parent, and attaching it to the object that exports?